FuelSDK-CSharp/ETClient.cs

```diff
@@ -248,7 +241,7 @@ public void RefreshToken(bool force = false)
             {
                 string json;
                 if (!string.IsNullOrEmpty(RefreshKey))
-                    json = string.Format("{{\"clientId\": \"{0}\", \"clientSecret\": \"{1}\", \"refreshToken\": \"{2}\", \"scope\": \"cas:{3}\" , \"accessType\": \"offline\"}}", configSection.ClientId, configSection.ClientSecret, RefreshKey, InternalAuthToken);
```

Comment:
@manivinesh 
Since we removed the usage of the InternalAuthToken attribute, we also removed the 
"scope" : "cas:<InternalAuthToken>" property from this json built. We tested in POSTMAN that a request without this
key:value pair works. Do you have any knowledge on this type of specific authentication - is this still used somewhere; can we consult some documentation?